### DISCUSSION TITLE
Undefined array key 'type' | FieldArgument.php

### DISCUSSION THREAD

**Discussion by EricChambersPowerDigital** [rebing/graphql-laravel#913] [Q&A]
**Thrown Error:** ErrorException: Undefined array key &quot;type&quot; in file /var/www/html/vendor/webonyx/graphql-php/src/Type/Definition/FieldArgument.php on line 89

**Error Code Location:**
<img width="821" alt="Screen Shot 2022-04-25 at 10 20 53 AM" src="https://user-images.githubusercontent.com/104379378/165131344-a992e8b9-afea-45ca-91e6-4a1c5a7b63b1.png">

**Console Log of Data Issue:**
<img width="1039" alt="Screen Shot 2022-04-25 at 10 22 41 AM" src="https://user-images.githubusercontent.com/104379378/165131756-f0533646-33cc-4e7e-8bb3-325e3732742a.png">

**Composer Package Versions:**
- "php": "^8.0"
- "laravel/framework": "^8.83"
- "webonyx/graphql-php": "^14.6.4"
- "rebing/graphql-laravel": "^8.2"

**Hack that fixes issue:**
<img width="1220" alt="Screen Shot 2022-04-25 at 10 24 48 AM" src="https://user-images.githubusercontent.com/104379378/165132012-ac08ff75-86fa-4009-aae0-c7f16ed5f9b8.png">

**Thought Process & More Information:**
The type definition is coming from the DB model class, the data type is the same as many other data types that still define the index as 'type'. I cannot see anything wrong with the model definition, relationships or anything in the code itself. Seems that the definition is some how being built incorrectly. 

Any help would be much appreciated. Thank you.

**Comment by mfn** [comment]
Please paste the definition where you defined `contact_id`

**Reply by EricChambersPowerDigital** [reply]
Thank you for your response and apologizes for the duplicate, was unsure what place was more appropriate. 

The definition comes directly from the database column named `contact_id` in our DB model class shown below. 

<img width="716" alt="Screen Shot 2022-05-05 at 3 16 11 PM" src="https://user-images.githubusercontent.com/104379378/167027178-01a57142-90c8-42f4-8ad7-9c66b8abe5d3.png">
<img width="732" alt="Screen Shot 2022-05-05 at 3 16 28 PM" src="https://user-images.githubusercontent.com/104379378/167027195-744bd8a3-9102-4955-9722-0199d305662e.png">

Please let me know if you need any more details.

**Reply by mfn** [reply]
Thank, but this is the Model: I wanted to see the _GraphQL_ type definition.

Also, please no screenshots: paste them as text and enclose them in tripple backticks!
